addons/api/tests/unit/serializers/storage-bucket-test.js

```diff
@@ -222,6 +255,62 @@ module('Unit | Serializer | storage bucket', function (hooks) {
     assert.deepEqual(serializedRecord, expectedResult);
   });
 
+  test('it serializes when updateing a MinIO (static credential by default) storage bucket correctly', async function (assert) {
```

Comment:
I am thinking on adding a test were we update an AWS storage bucket to be MinIO, but not sure that even makes sense, I will check with stakeholders and if does sense I will add such test.

Reply:
After checking with BE, the plugin type MUST be set at creation time ONLY. Can not be edited.